src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -21,6 +29,54 @@ type Props = {
 export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
   render() {
     const meta = this.props.profile.meta;
+    const overheads: ?(ProfilerOverhead[]) = this.props.profile
+      .profilerOverhead;
+    const calculatedStats: Map<string, ProfilerStats> = new Map();
+    // These 3 values only have single values, so we are not using ProfilerStats class for them.
+    let overheadDurations = 0;
+    let overheadPercentage = 0;
+    let profiledDuration = 0;
+    let totalSamplingCount = 0;
+
+    // Older profiles(Before FF 70) don't have any overhead info. Don't show anything if
+    // that's the case.
+    if (overheads) {
+      // Overhead keys that have min/max/mean values to loop.
+      const statKeys = [
+        'Overhead',
+        'Cleaning',
+        'Counter',
+        'Interval',
+        'Lockings',
+      ];
```

Comment:
They are always the same, you can extract them and also maybe derive a type. Although we can't derive a type from an array but rather from an object (see tabs-handling.js as an example). So this is an optional request.
But the type would make it possible to use it as key in `Map<string, ProfilerStats>` above :)
Maybe you can define the type "manually".